test/spec/ol/control/scaleline.test.js

```diff
@@ -265,6 +265,40 @@ describe('ol.control.ScaleLine', function() {
       var innerHtml4326 = ctrl.element_.innerHTML;
       expect(innerHtml4326).to.not.be(innerHtml3857);
     });
+
+    it.only('Projection\'s metersPerUnit affect scale for non-degree units', function() {
```

Comment:
`only` should be removed

Reply:
good call :)

Reply:
Good that our coverage check fails when an `.only` is accidentally left in the test code.